Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM node:12.18.4
+
+WORKDIR /usr/src/app
+
+COPY package.json .
+
+RUN npm install
+
+RUN npm install pm2 -g
```

Comment:
curious, what is pm2? thanks!

Reply:
pm2 is a process manager that ensures an application restarts if it closes for some reason. It's not really needed here, but I was trying something out.